src/Defs.h

```diff
@@ -114,7 +114,7 @@ extern uint32_t MPS_MAX;
 const uint32_t MPS_DEFAULT         = 10*1000;
 const uint32_t REPLY_EVERY_DEFAULT = 100;
 
-const uint32_t TEST_START_WARMUP_MSEC = 50;
+const uint32_t TEST_START_WARMUP_MSEC = 400;
```

Comment:
@AlaAyesh it increases real time of the test for 0.5sec as I understand. So using default `-t` argument it requires 1.5sec instead of 1.05sec. I am not confident that additional time is excluded from final calculation.